runtime/doc/lsp.txt

```diff
@@ -454,30 +454,44 @@ You probably want these inside a |ColorScheme| autocommand.
 Use |LspTokenUpdate| and |vim.lsp.semantic_tokens.highlight_token()| for more
 complex highlighting.
 
-The following groups are linked by default to standard |group-name|s:
```

Comment:
Why remove the link to `group-name` ?

Reply:
Because that link is no longer relevant to the list; the meaning has changed (similarly to the treesitter group list).

Reply:
Here are some more points:
- Technically, they now directly link to tree-sitter groups. Which won't be a valuable knowledge in on itself, in my opinion, as tree-sitter groups themselves are linked.
- This information feels like an implementation detail. And can be get by the user with `:Inspect` over the group name itself (as is described before the list).

Reply:
ok, let's add that to the commit message

Reply:
Done.